packages/hdwallet-core/package.json

```diff
@@ -26,5 +26,6 @@
   "devDependencies": {
     "@types/google-protobuf": "^3.15.1",
     "@types/lodash": "^4.14.168"
-  }
+  },
+  "gitHead": "27297629e92051d6442f056b0da56dd8a915a0f2"
```

Comment:
remove the `gitHead`s from all of the package jsons

Reply:
e8d0d107